### PR TITLE
audiofile: apply CVE patches

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

:hash: yikes

### CODE REVIEW

**Comment by DomT4** [comment]
I'll take a break at this point so ILZ can do whatever he's trying to do with CI 😄.

**Comment by ilovezfs** [comment]
hopefully it's OK now :crossed_fingers:

File: Formula/audiofile.rb [hunk 1]
```diff
@@ -28,6 +41,33 @@ class Audiofile < Formula
 
   depends_on "lcov" => :optional
 
+  # These have all been reported upstream but beside
+  # 03_CVE-2015-7747 not yet merged or fixed.
```
Comment (ilovezfs) [inline Formula/audiofile.rb]: comforting.

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
:headphones:

File: Formula/audiofile.rb (hunk 1 above)
Comment (DomT4) [inline Formula/audiofile.rb]: It's quite an impressive list to have not been handled upstream yet 😓

**Pull request closed without merging**